src/Common/src/Interop/Unix/System.Net.Http.Native/Interop.Multi.cs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Http
+    {
+        [DllImport(Libraries.HttpNative)]
+        public static extern SafeCurlMultiHandle MultiCreate();
+
+        [DllImport(Libraries.HttpNative)]
+        private static extern CURLMcode MultiDestroy(IntPtr handle);
+
+        [DllImport(Libraries.HttpNative)]
+        public static extern CURLMcode MultiAddHandle(SafeCurlMultiHandle multi_handle, SafeCurlHandle easy_handle);
+
+        [DllImport(Libraries.HttpNative)]
+        public static extern CURLMcode MultiRemoveHandle(SafeCurlMultiHandle multi_handle, SafeCurlHandle easy_handle);
+
+        [DllImport(Libraries.HttpNative)]
+        public static extern CURLMcode MultiWait(
+            SafeCurlMultiHandle multi_handle,
+            int extraFileDescriptor,
+            [MarshalAs(UnmanagedType.Bool)] out bool isExtraFileDescriptorActive,
+            [MarshalAs(UnmanagedType.Bool)] out bool isTimeout);
```

Comment:
Also fine for now, but I think we should go through our shims and remove all adornments like this which are just restating the default.